dbms/src/Flash/Statistics/JoinImpl.cpp

```diff
@@ -47,11 +47,10 @@ void JoinStatistics::collectExtraRuntimeDetail()
         for (const auto & join_build_stream : join_execute_info.join_build_streams)
```

Comment:
use branch to distinguish it might make code more readable.

Reply:
ok, updated.